src/drake/parser.clj

```diff
@@ -657,12 +657,21 @@
         p/emptiness)]
    (dissoc prod :vars)))
 
+(defn- attach-exec-dir
+  [prod file-path]
+  (if-let [dir (dfs/get-directory file-path)]
+    (update-in prod [:steps]
+               (fn [steps]
+                 (mapv #(assoc % :exec-dir (.getPath dir)) steps)))
+    prod))
+
 (def call-or-include-helper
   "See call-or-include-line below"
   (p/complex
    [_ percent-sign
     directive  (p/semantics (p/alt (p/lit-conc-seq "include" nb-char-lit)
-                               (p/lit-conc-seq "call" nb-char-lit))
+                               (p/lit-conc-seq "call" nb-char-lit)
+                               (p/lit-conc-seq "new-directive" nb-char-lit))
```

Comment:
This obviously needs a proper name. Ideas?

Ideally this would've been called %include, and %include is renamed %inline, but of course that's not going to happen.